src/main/java/step2/model/WinningRule.java

```diff
@@ -0,0 +1,24 @@
+package step2.model;
+
+public enum WinningRule {
+    RULE_2(4, 50000),
+    RULE_3(5, 15000000),
+    RULE_1(3, 5000),
+    RULE_4(6, 2000000000);
```

Comment:
```suggestion
    RULE_4(6, 2_000_000_000);
```
다음과 같이 표현하여 가옥성을 확보할 수 있습니다!